packages/react-composites/src/composites/CallWithChatComposite/CustomButton.tsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { ControlBarButton } from '@internal/react-components';
+import React from 'react';
+import { generateCustomControlBarButtonStrings } from '../CallComposite/components/buttons/Custom';
+import {
+  CallControlDisplayType,
+  CustomCallControlButtonCallbackArgs,
+  CustomControlButtonProps
+} from '../CallComposite/types/CallControlOptions';
+import { CallWithChatCompositeIcon, CallWithChatCompositeIcons } from '../common/icons';
+/* @conditional-compile-remove(control-bar-button-injection) */
```

Comment:
Yes, cause it's only used below in line 155
`export const onFetchCustomButtonPropsTrampoline`
with onFetchCustomButtonpropsTrampoline being conditionally compiled.

If line 12 is not conditionally compiled then it causes a lint error as it's not being used otherwise.